src/components/modules/subscriptionsBanner/SubscriptionsBanner.tsx

```diff
@@ -0,0 +1,301 @@
+import React from 'react';
+import { css } from 'emotion';
+import { ThemeProvider } from 'emotion-theming';
+import { body, headline, textSans } from '@guardian/src-foundations/typography/cjs';
+import { neutral } from '@guardian/src-foundations/palette';
+import { from, until } from '@guardian/src-foundations/mq';
+import { space } from '@guardian/src-foundations';
+import { Button, LinkButton, buttonReaderRevenue } from '@guardian/src-button';
+import { brand } from '@guardian/src-foundations/themes';
+import { Logo } from './Logo';
+import { Close } from './Close';
+
+const banner = css`
+    display: flex;
+    justify-content: center;
+    box-sizing: border-box;
+    width: 100%;
+    background-color: #006d67;
```

Comment:
Is this colour available in the `src-foundations/palette`?

Reply:
No, it's not in there 😢

Reply:
Ok no problem, if it's used elsewhere and is part of the Subscriptions branding then I wonder whether we should add it as a "reader revenue" token, if such a thing exists? One to raise with @SiAdcock. I can't say I know of any other places this colour's used off the top of my head so hardcoding it here is fine by me for now.